makeDockerfile.ts

```diff
@@ -171,5 +171,11 @@ for (const proposal of readProposals()) {
   previousProposal = proposal;
 }
 
-const contents = blocks.join('\n');
-fs.writeFileSync('Dockerfile', contents);
+export function refreshDockerfile() {
+  const contents = blocks.join('\n');
+  fs.writeFileSync('Dockerfile', contents);
+}
+
+if (require.main === module) {
```

Comment:
I thought we used ES modules and node doesn't support this idiom in that context.

Does .ts not get compiled to ES modules?

Reply:
Node only uses ESM if `package.json` has `"type": "module"`. But there's no package.json here so it defaults to CJS